arch/x86/kernel/slaunch.c

```diff
@@ -5,7 +5,7 @@
  *
  * Author(s):
  *     Daniel P. Smith <dpsmith@apertussolutions.com>
- *
+ *     Garnet T. Grimm <garnetgrimm720@gmail.com>
```

Comment:
Your Author and Signed-off need to be by the same email address.

Reply:
fixed